src/main/java/com/cburch/logisim/vhdl/base/VhdlSimConstants.java

```diff
@@ -77,6 +77,8 @@ public boolean isHidden() {
   public static final String SIM_COMP_PATH = SIM_PATH + "comp/";
   public static final String SIM_TOP_FILENAME = "top_sim.vhdl";
   public static final String VHDL_COMPONENT_SIM_NAME = "LogisimVhdlSimComp_";
+  // FIXME: hardcoded path. The "../src/" asks for troubles!
+  public static final String VHDL_COMPILE_COMMAND = "vcom -reportprogress 300 -work work ../src/";
```

Comment:
Fine with me for the moment. Did you check that this works on Windows, too? I know that inside the Tcl shell paths with slash and backslash work in princible (if properly escaped).

Reply:
I just moved to the constants class, I'm not the author of this command. But I'm on Windows and had no issue with it